advanced_samples/Dataview_Wind/Jupyter/azure-pipelines.yml

```diff
@@ -0,0 +1,119 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '/advanced_samples/Dataview_Wind/Jupyter'
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '/advanced_samples/Dataview_Wind/Jupyter'
+
+schedules:
+  - cron: '0 0 * * 0'
+    displayName: Weekly build
+    branches:
+      include:
+        - master
+    always: true
+
+variables:
+  - template: '/miscellaneous/build_templates/variables.yml'
+  - name: projPath
+    value: '/advanced_samples/Dataview_Wind/Jupyter'
+  - name: CredFile
+    value: config.ini
+  - name: CredFileLibrary
+    value: product-readiness.sds_python.ini
+  - name: analysisProject
+    value: SDSDataviewJupyter
+
+  - name: bulkUploadprojPath
+    value: '/advanced_samples/Bulk_Upload/DotNet'
+  - name: bulkUploadFiles
+    value: '/advanced_samples/Bulk_Upload/SampleCollections/DataviewWind'
+  - name: bulkUploadCredFile
+    value: appsettings.json
+  - name: bulkUploadCredFileLibrary
+    value: product-readiness.BulkUpload_DataviewWindFarm.json
+
+jobs:
+  - job: Tests
+    strategy:
+      matrix:
+        linux:
+          imageName: ubuntu-latest
+        windows:
+          imageName: windows-latest
+    pool:
+      vmImage: $(imageName)
+    steps:
+      - template: '/miscellaneous/build_templates/config.yml'
+        parameters:
+          fileOriginal: $(CredFile)
+          fileLibrary: $(CredFileLibrary)
+          filePath: $(projPath)
+
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.6'
+          architecture: 'x64'
+        displayName: 'Specify Python version'
+
+      - template: '/miscellaneous/build_templates/config.yml'
+        parameters:
+          fileOriginal: $(bulkUploadCredFile)
+          fileLibrary: $(bulkUploadCredFileLibrary)
+          filePath: $(bulkUploadprojPath)/BulkUploader
+
+      # - bash: |
```

Comment:
Should this commented out stuff be deleted?

Reply:
I was leaving it because i decided that maybe we don't need to upload and delete everytime (can keep it in OCS).  It could be deleted, but left it in case we needed to reupload/delete at some point (maybe I should I put it on a variable trigger??)